src/sota_tools/garage_push.cc

```diff
@@ -147,6 +151,27 @@ int main(int argc, char **argv) {
         return EXIT_FAILURE;
       }
     }
+
+    if (manifest_path != "") {
+      std::string manifest_json;
+      manifest_json = xml2json(Utils::readFile(manifest_path).c_str());
```

Comment:
My understanding is that `xml2json` can throw an exception, if it is correct then such exception won't be caught.